packages/tools/replay-tool/src/unknownChannel.ts

```diff
@@ -0,0 +1,122 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { FluidDataStoreRuntime, ISharedObjectRegistry } from "@fluidframework/datastore";
+import {
+    IFluidDataStoreRuntime,
+    IChannelFactory,
+    IChannelAttributes,
+    IChannelServices,
+    IChannel,
+} from "@fluidframework/datastore-definitions";
+import { ISequencedDocumentMessage, SummaryType } from "@fluidframework/protocol-definitions";
+import {
+    IFluidDataStoreContext,
+    IGarbageCollectionData,
+    IChannelSummarizeResult,
+} from "@fluidframework/runtime-definitions";
+
+class UnknownChannel implements IChannel {
+    constructor(
+        public readonly id: string,
+        public readonly attributes: IChannelAttributes,
+        services: IChannelServices)
+    {
+        services.deltaConnection.attach({
+            process: (message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown) => {
+            },
+            setConnectionState: (connected: boolean) => {
+            },
+            reSubmit: (content: any, localOpMetadata: unknown) => {
+            },
+            applyStashedOp: (content: any) => {
+            },
+        });
+    }
+
+    get IFluidLoadable() { return this; }
+    get handle(): IFluidHandle {
+        throw new Error("not implemented");
+    }
+
+    public summarize(fullTree?: boolean, trackState?: boolean): IChannelSummarizeResult {
+        return {
+            gcData: { gcNodes: { "/": [] } },
+            stats: {
+                treeNodeCount: 0,
```

Comment:
```suggestion
                treeNodeCount: 1,
```
For this tree node itself.